src/main/java/net/dv8tion/jda/api/events/interaction/SlashCommandEvent.java

```diff
@@ -83,4 +83,18 @@ public long getCommandIdLong()
     {
         return commandInteraction.getOptions();
     }
+
+    /**
+     * Gets the slash command String for this slash command.
+     * <br>This is similar to the String you see when clicking the interaction name in the client.
+     *
+     * <p>Example return for an echo command: {@code /say echo phrase: Say this}
+     *
+     * @return The command String for this slash command
+     */
+    @Nonnull
+    public String getText()
```

Comment:
This should be in the CommandInteraction interface and should have a more logical name than "text"

Reply:
You should make sure that this string is actually equivalent to the one in the client, right now I'm fairly certain that is not the case.

Reply:
Moved and resolved